test/games/fusion/gui/test_fusion_cosmetic_patches_dialog.py

```diff
@@ -41,3 +43,26 @@ def test_color_space(skip_qtbot):
     dialog.color_space_combo.setCurrentIndex(0)
     # Assert
     assert dialog.cosmetic_patches == FusionCosmeticPatches(color_space=ColorSpace.Oklab)
+
+
+@pytest.mark.parametrize(
+    ("music_start_value", "option_to_click", "music_end_value"),
+    [
+        (False, "stereo_option", True),
+        (True, "mono_option", False),
+    ],
+)
+def test_change_music_option(skip_qtbot, music_start_value: bool, option_to_click: str, music_end_value: bool) -> None:
+    cosmetic_patches = FusionCosmeticPatches(stereo_default=music_start_value)
+
+    dialog = FusionCosmeticPatchesDialog(None, cosmetic_patches)
+    skip_qtbot.addWidget(dialog)
+    str_to_option_map = {
+        "stereo_option": dialog.stereo_option,
+        "mono_option": dialog.mono_option,
+    }
```

Comment:
you should be able to use `getattr` for this. `getattr(dialog, option_to_click)`.